src/server/services/list/helpers/build-services-table-data.test.js

```diff
@@ -146,52 +117,45 @@ describe('#buildServicesTableData', () => {
 
       beforeEach(async () => {
         // return no services
-        nock(deployableServicesEndpointUrl.origin)
-          .get(deployableServicesEndpointUrl.pathname)
-          .query({ service: 'new-service-two' })
-          .reply(200, [])
-
-        // return 'new-service-two' status only
-        nock(inProgressServicesEndpointUrl.origin)
-          .get(inProgressServicesEndpointUrl.pathname)
-          .query({ kind: 'microservice', service: 'new-service-two' })
-          .reply(200, [
-            inProgressStatusFixture.find(
-              (item) => item.repositoryName === 'new-service-two'
+        nock(entitiesEndpointUrl.origin)
+          .get(entitiesEndpointUrl.pathname)
+          .query({ type: 'Microservice', name: 'forms-service' })
+          .reply(
+            200,
+            entityServicesFixture.filter(
+              (item) => item.name === 'forms-service'
             )
-          ])
+          )
 
         result = await buildServicesTableData({
-          service: 'new-service-two',
+          service: 'forms-service',
           isAuthenticated: true,
           userScopeUUIDs
         })
       })
 
-      test('With single result, Should provide all filters', () => {
+      test('With single result, should provide all filters', () => {
```

Comment:
your slowly getting all my capital `Shoulds` 🤣

Reply:
next i'll work on your your => you're :trollface: